docs/src/bootstrap.md

```diff
@@ -520,6 +520,40 @@ spec:
           maxParallel: 8
 ```
 
+#### Update application database password (recovery)
+
+By default, the user credentials for the application database are preserved. You can update 
+the application user password with additional configuration. You can generate your passwords, 
+store them as secrets, and update the database use your own secrets after recovery. Or you can 
+also let the operator generate a secret with randomly secure password for use.
+Please reference the  ["Bootstrap an empty cluster"](#bootstrap-an-empty-cluster-initdb) section
+for more information about secrets.
+
+The following example update the application user password with supplied secret `app-secret`
```

Comment:
I think we should explain here how this will affect bootstrap of replica clusters. The same consideration applies to the `pg_basebackup` bootstrap method.

If we change something after having restored a database, the cluster won't be able to reattach to the master.

Reply:
Maybe we should disable the synchronization with PostgreSQL content if the `replica` mode is enabled.

Reply:
This is a good call. When `replica` is active, this throws an error. Also, while in most cases when replica mode is enabled we don't want to have this code running (as it might have been performed on the primary), in some cases this could be desired (see when you import a database from outside Kubernetes in CloudNativePG). So, question:

- can we skip for now if in replica mode - and properly document it?
- can we open a new ticket to add a test for this case
- can we open a new ticket to manage reconciliation of the app database and user when the instance is promoted?

Thanks,
Gabriele

Reply:
yes, I'd like to have both: blocking it from the webhook and disable the synchronization on replica mode